src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        int randomNumber = new Random().nextInt(100);
+        ColorSupplier fff = new ColorSupplier();
+        Ball bigball = new Ball();
```

Comment:
According to our checklist, don't create temporary variables if you can directly return value from the method. It would be better to return a new Ball instance directly from the method without storing it in a variable.